app/ids-text/example.html

```diff
@@ -12,6 +12,7 @@
     <ids-text font-size="24">Size 24 (lg)</ids-text>
     <ids-text font-size="20">Size 20</ids-text>
     <ids-text font-size="16">Size 16 (base)</ids-text>
+    <ids-text font-size="16" font-weight="bold">Size 16 + Bold</ids-text>
```

Comment:
```suggestion
    <ids-text font-size="16" font-weight="bold">Size 16 + Bold</ids-text>
```

Lets remove this from this example as it throws it off. Can either be a separate page or no example (example is where you end up really using it).